packages/paste-core/components/tooltip/stories/index.stories.tsx

```diff
@@ -170,3 +171,32 @@ export const StateHookTooltip = (): React.ReactNode => {
 StateHookTooltip.story = {
   name: 'State hook Tooltip',
 };
+
+export const CustomizedTooltip = (): React.ReactNode => {
+  return (
+    <CustomizationProvider
+      baseTheme="default"
+      elements={{
+        TOOLTIP: {
+          backgroundColor: 'colorBackgroundErrorWeakest',
+          borderColor: 'colorBorderDestructive',
+          maxWidth: 'size10',
+          textAlign: 'center',
+        },
+        TOOLTIP_TEXT: {
+          color: 'colorTextErrorStrong',
+        },
+      }}
+    >
+      <Box as="div" minHeight="400px">
+        <Tooltip visible text={text('text', 'Welcome to Paste!')}>
+          <Button variant="primary">Open tooltip</Button>
+        </Tooltip>
+      </Box>
+    </CustomizationProvider>
+  );
+};
+
+CustomizedTooltip.story = {
+  name: 'Customized Tooltip',
+};
```

Comment:
Don't we usually have a story to VRT a custom element name too?

Reply:
I wasn't sure but just added one to tooltip and will keep that pattern in the future. Thanks!